docs/core/extensions/snippets/logging/logging.sln

```diff
@@ -19,6 +19,18 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "logger-message-generator",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "library-authors", "library-authors\library-authors.csproj", "{C0C79D16-5ECB-4B52-9303-15DCF6B0B639}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "getting-started", "getting-started\getting-started.csproj", "{AA405DF9-EA0A-4130-BD42-F662CFFFA0B1}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "getting-started-logger-message", "getting-started-logger-message\getting-started-logger-message.csproj", "{1179C5BB-3C2F-4253-9FCC-528DACF1D0D6}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "getting-started-type-category-name", "getting-started-type-category-name\getting-started-type-category-name.csproj", "{8CE47011-B4BB-4054-A48F-30CCF5AE5671}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "getting-started-open-telemetry", "getting-started-open-telemetry\getting-started-open-telemetry.csproj", "{D711C802-982D-4712-B70D-A3179C4313CC}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "aspnet-mvc", "aspnet-mvc\aspnet-mvc.csproj", "{D169B00F-66E9-4500-879D-487FE0E09F23}"
```

Comment:
I'm not really a fan of the MVC example, could I convince you to instead use an ASP.NET Core Minimal API instead? It adds a lot less file noise here, and it conveys the same teachings.

Reply:
Yeah, I felt bad about the large number of files. What if I converted it to inline source in the .md rather than a separate project?

 The challenge I hit with minimal API is that it doesn't naturally lead to using separate classes that will have constructor parameters. I can do method parameter injection instead, but then I don't get unique class names to show as the ILogger\<T> type parameter. Alternately I can create custom classes and register them in a DI container for minimal API, but then it felt like I need to show the extra DI registration and routing hookup or its unclear how the class I am introducing relates whereas MVC controller has a strong convention around it automatically.

Reply:
> offers a more streamlined solution with fewer components in motion.

Its more streamlined in the size of the full project, but needs more lines of text shown for folks reading the main doc. That isn't how I was prioritizing it before but I'll switch around based on your preference.

Reply:
switched, mvc is gone and minimal API is added instead.